src/consts.js

```diff
@@ -1,17 +1,36 @@
+const path = require("path");
+
 const TEMPLATES_NAME = [
     'apify_project',
     'basic_crawler',
     'cheerio_crawler',
     'hello_world',
     'puppeteer_crawler',
-    'puppeteer_single_page'
+    'puppeteer_single_page',
+];
+
+const EXAMPLES_NAME = [
```

Comment:
EXAMPLE_NAMES

Reply:
The `EXAMPLE_NAMES` property is not actually used, so removing it...